src/App.js

```diff
@@ -1,12 +1,67 @@
-import React from 'react';
+// Ik vind dit nog erg lastige materie, met name voor wat betreft het aanroepen en weergeven van de juiste endpoints. Ik heb hier dan ook hulp bij gehad van de SME ér en een medestudent, en het werkt nu. Ik heb echter niet het gevoel dat ik overal werkelijk doorgrond wat er gebeurt.
+
+import React, { useState, useEffect } from 'react';
 import './App.css';
+import axios from 'axios'
+// import pokemonLogo from './assets/pokemonLogo.svg'
+// Als ik het logo probeer weer te geven krijg ik een foutmelding die ik niet begrijp, dus om de applicatie te kunnen runnen, heb ik dit maar uitgezet.
+import Button from './components/button/Button';
+import PokemonCard from './components/pokemon-card/PokemonCard';
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemons, setPokemons] = useState([]);
```

Comment:
Als `pokemon` een object zal worden, dan zou ik een default waarde van `undefined` pakken. Nu lijkt het alsof het een array is en zal blijven, maar dat is niet zo
```suggestion
    const [pokemons, setPokemons] = useState();
```